### PR TITLE
Add example to README

### PR DESCRIPTION
**Pull request by sethladd**

### CODE REVIEW

**Comment by sethladd** [comment]
cc @lrhn

File: README.md [hunk 1]
```diff
@@ -2,6 +2,26 @@
...
 Reading data from package contents and files.
 
+A resource is data that can be read into a Dart program at runtime.
+A resource is identified by a URI. It can be loaded as bytes or data.
+The resource URI may be a package: URI.
+
+Example:
+
+```
+import 'package:resource/resource.dart';
```
Comment (lrhn) [inline README.md]: Technically you should also import dart:collection to have UTF8 for this to be a compete example. I'm not sure if that's important or not.

Comment (sethladd) [inline README.md]: ah, good call. will add. (meant dart:convert I assume?)

**Comment by sethladd** [comment]
OK, ptal. Thanks!

File: README.md (hunk 1 above)
Comment (lrhn) [inline README.md]: Yeah, that's what I said ;)

**Comment by lrhn** [comment]
Looks good, but could you squash the commits into one?

**Comment by sethladd** [comment]
Thanks!

**Pull request merged**